### ISSUE TITLE
specific Homebrew images not linking with notes. 

### ISSUE THREAD

**Issue by DaniSmol** [ebullient/ttrpg-convert-cli#431]
Certain Homebrew images &/or tokens are not linked with their corresponding notes. The notes themselves are being generated properly.

Notes are generated without images. 

## Configuration

- **OS**: Windows
- **CLI version**: ttrpg-convert-cli-2.3.10
- **CLI type**: cmd

**Sources**:
- ToB1, ToB2, ToB3
- Fantasy 5e Template
- DiceRoller enabled

Note- I am wondering if its possible that the Json for ToB and ToB2 image sources (URL) are the issue. ToB3 has simple spaces in the names for links where ToB and ToB2 use the %20 spacing. 
The ToB3 creatures are linking properly while 1 and 2 are not. 
Also I'd like to note that ToB3 does not have a Width and Height statistic under image URLs where ToB1 and 2 do. 

Runtime Command - ttrpg-convert.exe --index -o ToB_Test -c ConfigToB.json 5etools-mirror-2.github.io/data

Note - The config files directories are set for my system so don't forget to change them if needed.

[ttrpg-convert_ToB.out.txt.txt](https://github.com/ebullient/ttrpg-convert-cli/files/15070693/ttrpg-convert_ToB.out.txt.txt)
[Kobold Press; Tome of Beasts 2.json](https://github.com/ebullient/ttrpg-convert-cli/files/15070694/Kobold.Press.Tome.of.Beasts.2.json)
[Kobold Press; Tome of Beasts 3.json](https://github.com/ebullient/ttrpg-convert-cli/files/15070695/Kobold.Press.Tome.of.Beasts.3.json)
[Kobold Press; Tome of Beasts.json](https://github.com/ebullient/ttrpg-convert-cli/files/15070696/Kobold.Press.Tome.of.Beasts.json)
[ConfigToB.json](https://github.com/ebullient/ttrpg-convert-cli/files/15070697/ConfigToB.json)

**Comment by ebullient** [comment]
I think this has been fixed.. 🤞 
Can you try again with .12 at a minimum?

**Comment by DaniSmol** [comment]
Unfortunately Im not seeing a difference. Same behavior. I used the latest .12 version. Same config and settings. I used the 2.3.12 examples as well just to be sure it wasn't updated in them. 
[ttrpg-convert.4.29.2024.out.txt.txt](https://github.com/ebullient/ttrpg-convert-cli/files/15157125/ttrpg-convert.4.29.2024.out.txt.txt)
Heres the latest Config. 

Still has oddly enough the same behavior with many homebrew working including ToB3 but not ToB 1 or 2.
